lib/lambda_ethereum_consensus/store/state_db.ex

```diff
@@ -31,29 +32,30 @@ defmodule LambdaEthereumConsensus.Store.StateDb do
   end
 
   def prune_states_older_than(slot) do
+    Logger.info("[StateDb] Pruning started.", slot: slot)
     last_finalized_key = slot |> root_by_slot_key()
 
     with {:ok, it} <- Db.iterate(),
          {:ok, @stateslot_prefix <> _slot, _value} <-
            Exleveldb.iterator_move(it, last_finalized_key),
          {:ok, slots_to_remove} <- get_slots_to_remove(it),
          :ok <- Exleveldb.iterator_close(it) do
-      slots_to_remove |> Enum.map(&remove_by_slot/1)
+      slots_to_remove |> Enum.each(&remove_by_slot/1)
+      Logger.info("[StateDb] Pruning finished. #{length(slots_to_remove)} slots removed.")
     end
   end
 
   defp get_slots_to_remove(slots_to_remove \\ [], iterator) do
```

Comment:
can you add typespec here? I am confused if slots are integers or binaries

Reply:
Maybe I can create an issue to refactor this for clarity.